src/lib/authserver/ui/new_dir_inst.rb

```diff
@@ -142,7 +146,7 @@ def ok_handler
     end
 
     UI.ReplaceWidget(Id(:busy), Empty())
-    Popup.Message(_('New instance has been set up! Log output may be found in /var/log/YaST/y2log'))
+    Popup.Message(_('New instance has been set up! Log output may be found in %{y2log}'), y2log: y2log)
```

Comment:
Missing [`format`](https://ruby-doc.org/core-3.1.2/Kernel.html#method-i-format)

```suggestion
    Popup.Message(format(_('New instance has been set up! Log output may be found in %{y2log}'), y2log: y2log))
```